Auth0/SafariWebAuth.swift

```diff
@@ -88,6 +102,11 @@ class SafariWebAuth: WebAuth {
             else {
                 return callback(Result.failure(error: WebAuthError.noBundleIdentifierFound))
         }
+        if let response = self.parameters["response_type"], response.contains(ResponseType.id_token.rawValue) {
```

Comment:
`response_type` is mandatory and we should combine the if and guard in a single guard, it serves no purpose having them separated